packages/cli/src/commands/start.ts

```diff
@@ -173,10 +173,10 @@ export class Start extends BaseCommand {
 		await super.init();
 		this.activeWorkflowRunner = Container.get(ActiveWorkflowRunner);
 
-		await this.initLicense();
-
 		await this.initOrchestration();
 		this.logger.debug('Orchestration init complete');
+		await this.initLicense();
```

Comment:
Initialize `OrchestrationService` first so that `License` can initialize `LicenseManager` based on instance type.